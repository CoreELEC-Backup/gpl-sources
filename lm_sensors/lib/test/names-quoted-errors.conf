chip "blah-*"

# keyword with two quoted names, no intermediate whitespace
label "in0""foo"

# missing closing quote before EOL
label "in1" "bar

# (correct label statement, just for separation)
label "in2" "baz"

# missing closing quote before immediate EOF
label "in3" "baz