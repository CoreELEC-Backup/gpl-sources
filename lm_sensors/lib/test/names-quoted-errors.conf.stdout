1: CHIP
1: NAME: blah-*
2: EOL
4: LABEL
4: ERROR
5: EOL
7: LABEL
7: NAME: in1
7: ERROR
8: EOL
10: LABEL
10: NAME: in2
10: NAME: baz
11: EOL
13: LABEL
13: NAME: in3
13: ERROR
13: EOL
13: EOF
